azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/style/ChatCompositeTheme.kt

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.presentation.style
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.ThemeMode
+import com.microsoft.fluentui.theme.token.AliasTokens
+
+@Immutable
+data class AcsChatColors(
+    val content: Color,
+    val component: Color,
+    val background: Color,
+    val messageBackground: Color,
+    val messageBackgroundSelf: Color,
+)
+
+@Immutable
+data class AcsChatTypography(
+    val title: TextStyle,
+    val body: TextStyle,
+    // Define additional custom typography styles as required by Figma
+)
+
+@Immutable
+data class AcsChatShapes(
+    val messageBubble: Shape
+)
+
+val LocalAcsChatColors = staticCompositionLocalOf {
+    AcsChatColors(
```

Comment:
Would dark theme colors be added here as well?

Reply:
@Mrayyan yes, but this was to serve as starting point, we can expand on the theming in a separate PR